safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -86,14 +92,36 @@ def __init__(self, parent=None, iface=None):
         self.parent = parent
         self.setupUi(self)
 
-        self.setWindowTitle(self.tr('PetaJakarta Downloader'))
+        title = self.tr('PetaBencana Downloader')
+        self.setWindowTitle(title)
 
         self.iface = iface
 
+        self.source = None
+
+        self.radio_button_group = QButtonGroup()
+        self.radio_button_group.addButton(self.radio_button_production)
+        self.radio_button_group.addButton(self.radio_button_development)
+
+        self.radio_button_group.setExclusive(True)
+        self.radio_button_production.setChecked(True)
+        self.populate_combo_box()
+
+        settings = QSettings()
+        developer_mode = settings.value('inasafe/developer_mode', False,
```

Comment:
For your example of code above, I agree, it's normal. But 'developer_mode' is a boolean already in the QSettings.
![screen shot 2017-02-16 at 17 19 57](https://cloud.githubusercontent.com/assets/1609292/23029920/5f55ccc6-f46c-11e6-8780-5ec5f1646832.png)
It will not go into the `not isinstance(value, expected_type)`. So it should be fine, no ?

But I agree, there is a side effect of this kind of conversion. If you want to make a patch in another PR, you are welcome.